lib/cocoapods/deintegrator.rb

```diff
@@ -54,7 +54,7 @@ def deintegrate_pods_libraries(target)
 
     def deintegrate_shell_script_phase(target, phase_name)
       phases = target.shell_script_build_phases.select do |phase|
-        phase.name && phase.name =~ /\A(\u{1F4E6}\s)?#{Regexp.escape(phase_name)}\z/
+        phase.name && phase.name =~ /#{Regexp.escape(phase_name)}\z$/
```

Comment:
The actual change. This uses the same logic as CocoaPods integrator, and checks sans prefix.